packages/community-jazz-vue/src/auth/useDemoAuth.ts

```diff
@@ -0,0 +1,43 @@
+import { DemoAuth } from "jazz-tools";
+import { computed, markRaw, ref, watch } from "vue";
+import { useAuthSecretStorage, useJazzContext } from "../composables.js";
+import { useIsAuthenticated } from "./useIsAuthenticated.js";
+
+export function useDemoAuth() {
```

Comment:
We can remove this auth provider.

It's legacy and doesn't make much sense with the current auth system

Reply:
This is done. Removed the tests as well.